mcv/patchwork.py

```diff
@@ -0,0 +1,150 @@
+"""Patchwork"""
+import os
+import os.path
+import json
+import requests
+import subprocess
+
+from functools import partial
+
+PATCHWORK_API_ENDPOINT = os.getenv('PATCHWORK_API_ENDPOINT',
+                                   "https://api.patchworksecurity.com/api/v1")
+DEFAULT_UUID_PATH = '/etc/patchwork/uuid'
+DPKG_QUERY_CMD = ["/usr/bin/dpkg-query", "-W",
```

Comment:
This could equally go in the `mcv.apt` module as a Python function, and then be called from this module.  Not a blocker.